components/cartes-impact/gagnants-perdants/gagnants-perdants-component.jsx

```diff
@@ -288,218 +287,217 @@ class GagnantsPerdantsCard extends PureComponent {
                 </Button>
               </center>
             </div>
-          ) : (
-            [
-              isLoadingEtat ? (
-                <center className={classes.buttonPosition}>
-                  <CircularProgress color="secondary" />
-                </center>
-              ) : (
-                <>
-                  <div className={classes.impactContainer}>
+          )}
+          {!isDisabledEtat && isLoadingEtat && (
+            <center className={classes.buttonPosition}>
+              <CircularProgress color="secondary" />
+            </center>
+          )}
+          {!isDisabledEtat && !isLoadingEtat && (
+            <>
+              <div className={classes.impactContainer}>
+                <div>
+                  <ArrowUpwardIcon
+                    classes={{ root: classes.styleIconPerdant }}
+                    fontSize="default"
```

Comment:
The `medium` size does not exist. Only `default`, `inherit`, `small` and `large`.